imageio/imageio-dds/src/main/java/com/twelvemonkeys/imageio/plugins/dds/DDSReader.java

```diff
@@ -83,6 +84,10 @@ int[] read(ImageInputStream imageInput, int imageIndex) throws IOException {
 
         // type
         DDSType type = getType();
+        if (type == DDSType.DXT10) {
+            this.dxt10Header = DX10Header.read(imageInput);
+            type = this.dxt10Header.getDDSType();
+        }
```

Comment:
Don't think we need the `this` here.

```suggestion
        if (type == DDSType.DXT10) {
            dxt10Header = DX10Header.read(imageInput);
            type = dxt10Header.getDDSType();
        }
```